tests/pyccel/test_pyccel.py

```diff
@@ -585,9 +635,11 @@ def test_headers(language):
 
     compile_pyccel(cwd, test_file, pyccel_commands)
 
-
-    lang_out = get_lang_output(get_exe(test_file))
-    assert int(lang_out)
+    if language == "python":
+        lang_out = get_python_output(get_exe(test_file, language, prog=True))
+    else:
+        lang_out = get_lang_output(get_exe(test_file, language))
```

Comment:
We seem to have this output a lot. I wonder if we should combine these functions into a `get_output` function which receives the language as an argument. What do you think?